internal/config/fixtures/config.validate.yml

```diff
@@ -61,6 +61,12 @@ notif:
     client: "diun"
     topic: "docker/diun"
     qos: 0
+  ntfy:
+    endpoint: https://ntfy.sh
+    topic: diun-acce65a0-b777-46f9-9a11-58c67d1579c4
+    priority: default
+    timeout: 10s
+    emoji: package
```

Comment:
Also for consistency, `emoji` should be placed before `timeout`. And like my previous comments renamed `tags` and be an array.